mobilitypath_publisher/launch/mobilitypath_publisher.launch

```diff
@@ -0,0 +1,19 @@
+<!--
+  Copyright (C) 2018-2020 LEIDOS.
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<!--
+  This file is used to launch a the Mobility Path Publication Node
+-->
+<launch>
+    <rosparam command="load" file="$(find mobilitypath_publisher)/config/parameters.yaml"/>
```

Comment:
This should be nested inside the <node> tag so that it loads the parameters as private parameters.

Reply:
fixed